plugins/rbenv/rbenv.plugin.zsh

```diff
@@ -1,60 +1,69 @@
-_homebrew-installed() {
-  type brew &> /dev/null
-}
-
-FOUND_RBENV=0
-rbenvdirs=("$HOME/.rbenv" "/usr/local/rbenv" "/opt/rbenv" "/usr/local/opt/rbenv")
-if _homebrew-installed && rbenv_homebrew_path=$(brew --prefix rbenv 2>/dev/null); then
-    rbenvdirs=($rbenv_homebrew_path "${rbenvdirs[@]}")
-    unset rbenv_homebrew_path
-    if [[ $RBENV_ROOT = '' ]]; then 
-      RBENV_ROOT="$HOME/.rbenv"
+# This plugin loads rbenv into the current shell and provides prompt info via
+# the 'rbenv_prompt_info' function.
+
+FOUND_RBENV=$+commands[rbenv]
+
+if [[ $FOUND_RBENV -ne 1 ]]; then
+    rbenvdirs=("$HOME/.rbenv" "/usr/local/rbenv" "/opt/rbenv" "/usr/local/opt/rbenv")
+    for dir in $rbenvdirs; do
+        if [[ -d $dir/shims ]]; then
+            export PATH="$dir/shims:$dir/bin:$PATH"
```

Comment:
I don't think `$dir/shims` is necessary, `eval "$(rbenv init --no-rehash - zsh)"` should take care of this.

Reply:
True - I checked the init output, and you are right.
I'll remove this line.